src/pages/api/thirdparty/schema.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from 'graphql-tag'
+
+/* Data we will return to any third-party querying the API. */
+export const typeDefs = gql`
+  scalar JSON
+
+  enum ArticleTag {
```

Comment:
@gidjin @jcbcapps What do y'all think about creating a group of enums for approved tags to be passed to the getPublicArticles query?

Reply:
I'm good with this approach. My only question, which is in another comment, is around the fact that the API will take `CNOTE` and the tag is actually `C-Note`.